### ISSUE TITLE
Why are you also sampling batch norm parameters?

### ISSUE THREAD

**Issue by wclaus22** [noahfl/sdr-densenet-pytorch#3]
Looking at your implementation you're also sampling batch norm parameters? Is there any particular reason or intuition behind that? I don't think you mention batch norm in the article you posted to arxiv. Thank you.

**Comment by noahfl** [comment]
Batch norm layers create weights stored in the parameters field of the PyTorch model. After some experimenting we found a negligible difference between leaving in and omitting the batch norm weights and decided to just leave them in for the sake of simplicity.
